packages/app/src/runs/RunsContainer.vue

```diff
@@ -3,20 +3,22 @@
     v-if="!currentProject?.projectId || !cloudViewer?.id"
     :gql="props.gql"
   />
-  <RunsEmpty
-    v-else-if="!currentProject?.cloudProject?.runs?.nodes.length"
-    :gql="currentProject"
-  />
-  <div
-    v-else
-    data-cy="runs"
-  >
-    <RunCard
-      v-for="run of currentProject?.cloudProject?.runs?.nodes"
-      :key="run.id"
-      :gql="run"
+  <template v-else-if="currentProject?.cloudProject?.__typename === 'CloudProject'">
```

Comment:
Why do we need to check against the typename? Seems like a bit of a code smell - changing the name of a GQL would then break the front-end.

Reply:
And it should, since we chose to have the state be stored in the backend.